services/ui-src/src/components/reports/DrawerReportPage.tsx

```diff
@@ -137,16 +113,11 @@ export const DrawerReportPage = ({ route }: Props) => {
       <Heading as="h3" sx={sx.dashboardTitle}>
         {verbiage.dashboardTitle}
       </Heading>
-      {entities?.length ? (
-        entityRows(entities)
-      ) : (
-        <Text sx={sx.emptyEntityMessage}>
-          {missingEntitiesMessage}{" "}
-          <Link as={RouterLink} to={missingEntitiesLink.href}>
-            {missingEntitiesLink.text}
-          </Link>
-        </Text>
-      )}
+      <Box sx={sx.missingEntityMessage}>
+        {entities?.length
+          ? entityRows(entities)
+          : parseCustomHtml(verbiage.missingEntityMessage || "")}
```

Comment:
Note for reviewer: Changed this to just parse whatever custom html we pass it, similar to other text areas on form pages.